### DISCUSSION TITLE
Copilot for JetBrains does not support 2024.3 EAP

### DISCUSSION THREAD

**Discussion by OstAlex** [community/community#143243] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

![image](https://github.com/user-attachments/assets/4e088b91-593f-4808-8a11-3b1eecb53d36)

**Comment by samus-aran** [comment]
Hey @OstAlex 👋  Thanks for raising this. The team are currently working on this so do stay tuned for this update when available 👍

**Comment by samus-aran** [comment, marked as answer]
@OstAlex would you be able to try out the latest Copilot nightly build (Copilot 1.5.28.7452) with 2024.3 EAP? [Here is the link](https://plugins.jetbrains.com/plugin/17718-github-copilot/versions/nightly/629156) 👈 Let us know how you get on 👍

**Reply by OstAlex** [reply]
Geart ! it works !

**Reply by samus-aran** [reply]
Fabulous! Thanks for responding back @OstAlex ⭐
